packages/@sanity/components/src/selects/StatelessSearchableSelect.js

```diff
@@ -170,85 +174,88 @@ export default class StatelessSearchableSelect extends React.PureComponent {
+                tabIndex={0}
+                onKeyPress={disabled ? null : this.handleArrowKeyPress}
+              >
+                <FaAngleDown color="inherit" />
+              </div>
+            )}
+            {
+              isLoading && (
+                <Spinner />
+              )
+            }
           </div>
-        </div>
-        <div className={styles.stickyContainer} style={{width: `${this.props.width}px`}}>
-          {isOpen && (
-            <StickyPortal
-              isOpen={isOpen}
-              onResize={onResize}
-              onlyBottomSpace={false}
-              useOverlay={false}
-              addPadding={false}
-              scrollIntoView={false}
-            >
-              <Stacked>
-                {isActive => (
-                  <CaptureOutsideClicks onClickOutside={isActive ? onClose : null}>
-                    <Escapable onEscape={event => (isActive || event.shiftKey) && onClose()} />
-
-                    <div
-                      className={`
-                        ${isOpen ? styles.listContainer : styles.listContainerHidden}
-                        ${
-                          dropdownPosition === 'top'
-                            ? styles.listContainerTop
-                            : styles.listContainerBottom
+        </Target>
+        {
+          isOpen && (
+            <Stacked>
+              {isActive => (
+                <Portal>
+                  <Popper
+                    placement="bottom"
+                    className={styles.popper}
+                    modifiers={
+                      {
+                        preventOverflow: {
+                          boundariesElement: 'viewport'
                         }
-                        ${items.length === 0 ? styles.listContainerEmpty : ''}
-                      `}
-                      style={{width: `${this.props.width}px`}}
-                      ref={this.setListElement}
-                    >
-                      {items.length === 0 &&
-                        !isLoading && <p className={styles.noResultText}>No results</p>}
-                      {items.length === 0 &&
-                        isLoading && (
-                          <div className={styles.listSpinner}>
-                            <Spinner message="Loading items…" />
-                          </div>
-                        )}
-                      {items.length > 0 && (
-                        <SelectMenu
-                          items={items}
-                          value={value}
-                          onSelect={this.handleSelect}
-                          renderItem={renderItem}
-                          highlightIndex={highlightIndex}
-                        />
-                      )}
+                      }
+                    }
+                  >
+                    <div>
+                      <CaptureOutsideClicks onClickOutside={(isActive && isOpen) ? this.handleClose : undefined}>
```

Comment:
For consistency with other usages of this pattern: Use `(isActive && isOpen) ? this.handleClose : null` (`null` instead of `undefined`) here too